en/sql/tuning.rst

```diff
@@ -368,6 +385,228 @@ If you ensure that left table's row number is a lot smaller than the right table
+
+Selectivity
+--------------
+
+**Selectivity**/ is the ratio of data to be selected when a specific predicate is evaluated. The optimizer calculates the selectivity for each predicate in the **WHERE** clause. **CUBRID**/ assumes that the entire data is evenly distributed and calculates the **selectivity**/ using the **Number of Distinct Value**/ of statistics.
+
+::
+
+    SELECTIVITY = 1 / Number of Distinct Value
+
+.. code-block:: sql
+
+    CREATE TABLE t1 (code INT, name VARCHAR(20));
+    INSERT INTO t1 VALUES(1,'Park'),(2,'Park'),(3,'Park'),(4,'joo'),(5,'joo'),(5,'joo');
+    CREATE INDEX i_t1_code ON t1(code,name);
+    UPDATE STATISTICS ON t1;
+     
+    ;info stats t1
+
+::
+
+    CLASS STATISTICS
+    ****************
+     Class name: t1 Timestamp: Fri Dec 20 13:07:58 2024
+     Total pages in class heap: 1
+     Total objects: 6
+     Number of attributes: 2
+     Attribute: code (integer)
+        Number of Distinct Values: 5
+        B+tree statistics:
+            BTID: { 0 , 5760 }
+            Cardinality: 5 (5,5) , Total pages: 3 , Leaf pages: 1 , Height: 2
+     
+     Attribute: name (character varying)
+        Number of Distinct Values: 2
+
+.. code-block:: sql
+
+    ;plan detail
+    SELECT /*+ recompile */ * FROM t1 WHERE code = 3 AND name IN ('Song', 'Ham');
+
+::
+
+    Join graph terms:
+    term[0]: [dba.t1].code=3 (sel 0.2)
+    term[1]: [dba.t1].[name] range ('Ham' =  or 'Song' = ) (sel 0.75)
+
+**term[0]: [dba.t1].code=3**/ The **Number of Distinct Values**/ of **code**/ in the statistics is 5, so the selectivity is 1/5 or 0.2.
+**term[1]: [dba.t1].[name] range ('Ham' = or 'Song' = )** performs an **OR** operation on two values of name. Since the **Number of Distinct Values** for **name** is 2, the selectivity for each value is 1/2. Adding the two values and subtracting the intersection gives the calculation **0.5 + 0.5 - (0.5 x 0.5)**, resulting in 0.75.
+
+Expected number of rows
+-----------------------------
+
+The **expected number of rows**  is calculated by multiplying the **selectivity** by the **total number of data rows**.
+
+::
+
+    Number of expected rows = total rows of a table * selectivity
+
+.. code-block:: sql
+
+    ;plan detail
+    SELECT /*+ recompile */ * FROM t1 WHERE name = 'Park';
+
+::
+
+    Join graph nodes:
+    node[0]: dba.t1 dba.t1(6/1)
+    Join graph terms:
+    term[0]: [dba.t1].[name]='Park' (sel 0.5)
+     
+    Query plan:
+     
+    sscan
+        class: t1 node[0]
+        sargs: term[0]
+        cost:  1 card 3
+
+**dba.t1(6/1)** indicates that the total number of rows in the t1 table is 6 and the number of pages is 1. Since the selectivity of **[dba.t1].[name]='Park'**/ is 0.5, the expected number of rows is 3, which is calculated as **6 x 0.5**. **cost: 1 card 3**/ in the execution plan indicates that the cost is 1 and the expected number of rows is 3.
```

Comment:
```suggestion
**dba.t1(6/1)** indicates that the total number of rows in the t1 table is 6 and the number of pages is 1. Since the selectivity of **[dba.t1].[name]='Park'**\ is 0.5, the expected number of rows is 3, which is calculated as **6 x 0.5**. **cost: 1 card 3**\ in the execution plan indicates that the cost is 1 and the expected number of rows is 3.
```